index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
Maybe stick this in an `examples` directory to keep everything _not_ the provider easily separated?

Reply:
I think this still needs to get moved to examples?

Reply:
Done!